apps/backend/src/datasources/stfc/StfcProposalDataSource.ts

```diff
@@ -134,4 +137,64 @@ export default class StfcProposalDataSource extends PostgresProposalDataSource {
 
     return result;
   }
+
+  async getProposalsFromView(
+    filter?: ProposalsFilter,
+    first?: number,
+    offset?: number,
+    sortField?: string,
+    sortDirection?: string,
+    searchText?: string
+  ): Promise<{ totalCount: number; proposalViews: ProposalView[] }> {
+    const proposals = await super.getProposalsFromView(
```

Comment:
It is good that you call the parent class method here and it won't require a lot of additional changes or conflict resolutions against the multi-instrument proposals approach. 👍 💯